modules/exploits/windows/fileformat/office_excel_slk.rb

```diff
@@ -0,0 +1,116 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule  < Msf::Exploit::Remote
+  Rank = ManualRanking
+
+  include Msf::Exploit::Remote::HttpServer
+  include Msf::Exploit::FILEFORMAT
+  include Msf::Exploit::Powershell
+  include Msf::Exploit::EXE
+
+  def initialize(info = {})
+      super(update_info(info,
+          'Name' => "Microsoft Excel .SLK Payload Delivery",
+          'Description' => %Q{
+              This module generates a download and execute powershell
+              command to be placed in an .SLK Excel spreadsheet.
+              When executed, it will retrieve a payload via HTTP
+              from a web server.
```

Comment:
Should probably mention that the user will receive one or more warning popups when opening the file.